tests/unit/src/main/java/org/mule/tck/testmodels/fruit/Apple.java

```diff
@@ -141,4 +141,8 @@ public void setWashed(boolean washed) {
   public void setMuleContext(MuleContext context) {
     this.muleContext = context;
   }
+
+  public MuleContext getMuleContext() {
```

Comment:
Apple test class doesn't have any field annotated as Parameter, so it ended up falling back to bean properties, and the restriction that "every parameter should have a getter" fails for the mule context field.